src/utilities/colors.scss

```diff
@@ -62,10 +66,10 @@
 /* Set the text color to inherit */
 .text-inherit       { color: inherit !important; }
 
-// Text states
-// These can probably all be regular utilities
+// Pending states
+// This will be deprecated in the future, use .text-yellow instead
 .text-pending { color: $yellow-800 !important; }
-// Separate text and background colors in future to improve a11y
+// This will be deprecated in the future, use .bg-yellow-dark instead
```

Comment:
Added a warning with https://github.com/primer/css/pull/737/commits/227f9f53204f7d0d264f83c4b7f151352815054f. This will show up every time it gets compiled, right? Like the column warnings. Hmm.. yeah, that feels a bit aggressive. Like something went wrong.

Could #740 be used for `sentinel` or some other bot? Only warn in PRs when `.text-pending` gets added, would be 👌 .

Reply:
Ohh.. yes! That sounds much better. It guess it forces us to replace all the `.text-pending` at the same time we want to deprecate it? But that might be a good thing. :smile: We _should_ to do it at some point anyways.

Question: Should I remove the Sass `@warn` again?